src/game/game.py

```diff
@@ -5,7 +5,50 @@
 class Game(polymodel.PolyModel):
     """Represents a single instance of a Game"""
     date = ndb.DateTimeProperty(auto_now_add=True, required=True)
-    player_keys = ndb.KeyProperty(kind="User", repeated=True, required=True)
+    player_keys = ndb.KeyProperty(kind="User", repeated=True)
```

Comment:
I only see 2.

you could easily get rid of winners, but then you would lose the ability to query off of winners